src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TransmissionStorage.cs

```diff
@@ -33,8 +33,8 @@ public TransmissionStorage()
         {
             this.files = new ConcurrentQueue<IPlatformFile>();
             this.badFiles = new ConcurrentDictionary<string, string>();
-            TimeSpan ClearBadFilesInterval = new TimeSpan(29, 0, 0); // Arbitrarily aligns with IIS restart policy of 29 hours in case IIS isn't restarted.
-            this.clearBadFiles = new Timer((o) => this.badFiles.Clear(), null, ClearBadFilesInterval, ClearBadFilesInterval);
+            TimeSpan clearBadFilesInterval = new TimeSpan(29, 0, 0); // Arbitrarily aligns with IIS restart policy of 29 hours in case IIS isn't restarted.
```

Comment:
Doh, this one's on me, I was making a const originally and then changed the type and forgot to change the casing.